### DISCUSSION TITLE
Hardware requirements

### DISCUSSION THREAD

**Discussion by R0land013** [firefly-iii/firefly-iii#8301] [Firefly III]
Hi there. I would like to deploy firefly on a vps for my personal use. And I was wondering what are the system requirements of Firefly. So I can purchase a VPS that match the requirements, so I don’t spend more or less than what it’s needed.

**Comment by JC5** [comment]
Well, the demo site runs on a Standard_B1ms from Azure, so that should be fast enough. Check out the table to see how it compares:

https://learn.microsoft.com/en-us/azure/virtual-machines/sizes-b-series-burstable
